crates/viewer/re_space_view_spatial/src/transform_component_tracker.rs

```diff
@@ -20,11 +20,13 @@ use re_types::ComponentName;
 /// This is a huge performance improvement in practice, especially in recordings with many entities.
 #[derive(Default)]
 pub struct TransformComponentTracker {
-    /// Which entities have had any `Transform3D` component at any point in time.
-    transform3d_entities: IntSet<EntityPathHash>,
+    /// Which entities have had any `Transform3D` component at any point in time, and which
+    /// components they actually make use of.
+    transform3d_entities: HashMap<EntityPathHash, IntSet<ComponentName>>,
```

Comment:
Any reason the outer `IntSet` turned into a `HashMap`?

Reply:
Ahh, no. Thank you. Should have used IntMap.